crates/holochain/src/conductor/conductor.rs

```diff
@@ -1949,6 +1987,61 @@ mod app_status_impls {
                 .await?;
             Ok(disabled_app)
         }
+
+        /// Prepare module config overrides from app manifest.
+        ///
+        /// The config override is returned only if at least one override is needed.
+        pub(crate) fn space_config_override(
+            manifest: &AppManifest,
+        ) -> ConductorResult<Option<Config>> {
+            let config = Config::default();
```

Comment:
Will these defaults override what has been set in the conductor config, if different from the defaults?
E.g. in the conductor config's `advanced` field some value is set for `core_bootstrap: {backoff_min_ms: 1000}`, but the default is `1000 * 5`, I think that would get overridden like this.
Do you need to use the config settings from the conductor wide config here instead of `..Default::default()`?

Reply:
Good point!